File: demo09.pl
#!/usr/bin/perl -w

#Marking test
#Written by Raphael Rueda


print "Enter a mark for Raphael Rueda's tests", "\n";
$mark = <STDIN>;

if ($mark < 100) {
    print "Are you serious? Waaaaaay too low.", "\n";
} else {
    print "Acceptable.", "\n";
}

File: demo09.py
#!/bin/usr/python

#Marking test
#Written by Raphael Rueda

import sys

print "Enter a mark for Raphael Rueda's tests"
mark = int(sys.stdin.readline())

if mark < 100:
    print "Are you serious? Waaaaaay too low."
else:
    print "Acceptable."
